features/step_definitions/generator_steps.rb

```diff
@@ -28,6 +35,11 @@
   EOH
 end
 
+Given(/^I debug$/) do
```

Comment:
The advantage to putting it in the gemspec as a dev dep as opposed to in the gemfile is that it shows up on rubygems.org as a dev dep (gemfile deps dont, but still get pulled in)